rasa/core/training/story_reader/yaml_story_reader.py

```diff
@@ -343,7 +342,7 @@ def _parse_bot_message(self, step: Dict[Text, Any]) -> None:
             )
             return
 
-        self._add_event(bot_message, {"e2e_text": bot_message})
+        self._add_event(bot_message, {"e2e_text": bot_message}, is_e2e = True)
```

Comment:
@wochinge Correct me if I'm wrong... I think the `is_e2e` tag was supposed to be here. 
There was a test and it worked because `ActionExecuted.__eq__()` only compared action names and not whether there was `e2e_text` and,  whether it was the same. (updated that as well)

Reply:
The `is_e2e` should no longer be needed. We only had that for the hacky markdown format

Reply:
How will it know to set `e2e_text` in `ActionExecuted` if we don't pass `is_e2e` to it?

Reply:
yaml handles this here https://github.com/RasaHQ/rasa/blob/dfecdc7a6bff0bed20c54cd650860350667d6d80/rasa/core/training/story_reader/yaml_story_reader.py#L333

Reply:
ah! my bad! 
then I'll change `markdown` to work the same way and change it here so that the `action_name` is empty
and remove the `is_e2e` fluff we had before :)

Reply:
why do we keep the markdown stuff in the first place? Can't we just omit it's implementation?

Reply:
I mean we already have all of it done, so, why not?

Reply:
because 
1) it's not tested
2) we didn't consult anybody else how the format should look
3) Markdown is very likely gonna be dropped

Reply:
If you do a revert commit we can still re-add it later in case we need it, but for now I'd simply drop it.

Reply:
With respect to tests, I added one similar to the yaml one. 
I totally agree with 2 and 3!
The problem for me is that it means I (just like our users) have to manually rewrite all my stories to yaml to test anything I do with e2e now... a little bit of a pain 😅

Reply:
> The problem for me is that it means I (just like the rasa users) have to manually rewrite all my stories to yaml to test anything I do with e2e now... a little bit of a pain 😅

I feel you, but I am afraid there is no way around it as we have to do it anyway sooner or later